CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 **********
 
+[5.6.0] - 2024-01-24
+********************
+Changed
+=======
+* Added configurable client.id to base configuration
```

Comment:
1. Can you provide the setting name?
2. Here and/or in the docs, could you provide an example of how it is meant to be used?